src/core/namespace.c

```diff
@@ -116,6 +116,12 @@ typedef struct MountList {
         size_t n_mounts;
 } MountList;
 
+static const BindMount bind_journal_sockets_table[] = {
```

Comment:
Why not to use `MountEntry` ?

Reply:
That would be unnecessarily noise, as `source_const` and such must be spelled out. Plus, `append_static_mounts` doesn't take in source_const (bind mounts).

Both could be made to work ofc, but I like `BindMount` better.

Reply:
OK. That's not a strong opinion. Feel free to keep this as is. I just simply wonder why `MountEntry` is not used here. All other tables are written in `MountEntry`. So, personally I'd like to follow other tables.